PythonTool/TagExpansion.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import numpy as np
+import os
+from pymoab import core, tag, types
+
+
+def parse_arguments():
+    """
+    Parse the argument list and return a mesh file location.
+
+    Input:
+    ______
+       none
+
+    Returns:
+    ________
+       args: Namespace
+           User supplied mesh file location.
+    """
+
+    parser = argparse.ArgumentParser(description="Expand vector tags to scalar tags.")
+
+    parser.add_argument("meshfile",
+                        type=str,
+                        help="Provide a path to the mesh file."
+                        )
+
+    args = parser.parse_args()
+
+    return args
+
+
+def tag_expansion(mesh_file):
+    """
+    Expand the vector tags in the given mesh data file. Write a file to disk
+    for each time state with the corresponding scalar tag values from each vector tag.
+
+    Input:
+    ______
+       mesh_file: str
+           User supplied mesh file location.
+
+    Returns:
+    ________
+       none
+    """
+
+    # Load the mesh file to create a reference mesh.
+    mb2 = core.Core()
+    mb2.load_file(mesh_file)
+
+    # Retrieve an arbitrary MBHEX element in the mesh and extract the tag list.
+    root2 = mb2.get_root_set()
+    hexes2 = mb2.get_entities_by_type(root2, types.MBHEX)
+
+    if len(hexes2) == 0:
+        print("WARNING: No hex elements were found in this data file.")
+        exit()
+
+    tag_list2 = mb2.tag_get_tags_on_entity(hexes2[0])
+
+    # Check if each tag is a vector. If so, delete it.
+    for tag in tag_list2:
```

Comment:
@gonuke, @piperlincoln and I weren't sure if we'd want to remove all other tags from the file or only remove other vector tags from our reference mesh. Thoughts?